java/HealthVault Mobile/HVSample/src/main/res/menu/bottom_nav_items.xml

```diff
@@ -8,7 +7,7 @@
 	<item
 		android:id="@+id/menuActionPlan"
 		android:title="@string/menu_ActionPlan"
-		android:icon="@drawable/home_icon_select"/>
+		android:icon="@drawable/habits_icon_selected"/>
```

Comment:
Looks like we have all different styles for naming resources. Ids for items we create seem to be camelCase, but resource files have underscores, and the string resource above is a mix of the two with "menu_ActionPlan". We should be consistent across all naming conventions for resources.